NGitLab.Mock/Config/GitLabMergeRequest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NGitLab.Mock.Config
+{
+    /// <summary>
+    /// Describe a merge request in a GitLab project
+    /// </summary>
+    public class GitLabMergeRequest : GitLabObject<GitLabProject>
+    {
+        /// <summary>
+        /// Title (required)
+        /// </summary>
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Author username (required)
+        /// </summary>
+        public string Author { get; set; }
+
+        /// <summary>
+        /// Assignee username
+        /// </summary>
+        public string Assignee { get; set; }
```

Comment:
An merge request can have multiple assignees

Reply:
NGitLab does not implement multiple assignees for merge requests, only for issues